yubihsm/objects.py

```diff
@@ -1372,17 +1374,57 @@ def encrypt_ecb(self, data: bytes) -> bytes:
         :param data: The data to encrypt.
         :return: The encrypted data.
         """
-        msg = struct.pack("!H", self.id) + data
-        return self.session.send_secure_cmd(COMMAND.ENCRYPT_ECB, msg)
+
+        if len(data) % AES_BLOCK_SIZE != 0:
+            raise ValueError("Invalid argument")
```

Comment:
I'd use a more descriptive string here, like "Data length must be a multiple of 16". Same goes for the exceptions in the other functtions.

Reply:
I agree! I've pushed a commit that fixes that.